### PR TITLE
Feedback updates

### PR DESCRIPTION
**Pull request by sethun60**
The PR does the following

1. Renamed classnames to be more semantic
2. Removed redundant CSS
3. Added some new styling

### CODE REVIEW

File: password.html [hunk 1]
```diff
@@ -5,14 +5,14 @@
     <title>Password Strength Background</title>
   </head>
   <body>
-    <div class="inner-background">
+    <div class="content-container">
```
Comment (shycoder-06) [inline password.html:8]: why this name?

File: style.css [hunk 2]
```diff
@@ -1,38 +1,33 @@
...
-.box {
-  text-align: left;
-  margin-top: 16px;
-  margin-bottom: 16px;
+.description {
+  text-align: center;
+}
+
+.input-field {
+  margin: 16px 0;
```
Comment (shycoder-06) [inline style.css:19]: multi line test comment

File: style.css [hunk 3]
```diff
@@ -1,38 +1,33 @@
...
   background-color: #ffffff;
-  text-align: center;
   padding: 40px;
 }
 
-.box {
-  text-align: left;
-  margin-top: 16px;
-  margin-bottom: 16px;
+.description {
```
Comment (shycoder-06) [inline style.css:14]: trsgt

File: style.css [hunk 4]
```diff
@@ -1,38 +1,33 @@
...
+  text-align: center;
+}
+
+.input-field {
```
Comment (shycoder-06) [inline style.css:18]: fgsdfgsdfg

File: style.css [hunk 5]
```diff
@@ -1,38 +1,33 @@
...
+  margin: 16px 0;
 }
 
 .label {
   display: block;
-  padding-top: 8px;
-  padding-bottom: 8px;
-  text-align: left;
+  padding: 8px 0;
```
Comment (shycoder-06) [inline style.css:24]: rthfgh

**Review by shycoder-06 — CHANGES_REQUESTED** [review]

File: password.html [hunk 6]
```diff
@@ -5,14 +5,14 @@
...
       <h1>Image Password Strength</h1>
-      <p>Change the password to see the effect</p>
-      <div class="box">
+      <p class="description">Change the password to see the effect</p>
+      <div class="input-field">
```
Comment (shycoder-06) [inline password.html:11]: dghdfgh

File: password.html (hunk 1 above)
Comment (sethun60) [reply password.html:8]: why not this name?

**Review by shycoder-06 — APPROVED** [review]
LGTM

**Pull request merged**